default.hbs

```diff
@@ -87,19 +87,23 @@ $(document).ready(function () {
 
     // Filter
     $('.filter-selection').click(function(event) {
-        let classToFilter = event.target.innerHTML.replace(/\s+/g, '-').toLowerCase();
-        let hashToFilter;
-        if (classToFilter[0] == '#') {
-            hashToFilter = classToFilter.slice(1);
-        }
+        let classToFilter = event.target.innerHTML;
+        let internalTag = $(this).hasClass('internal-tag');
+        // Basic slug generator: replace whitespaces with dashes, then strip punctuation
+        classToFilter = classToFilter.replace(/\s+/g, '-').replace(/[.,'!?]/, '').toLowerCase();
```

Comment:
Could you add a TODO comment to this stating that the behaviour of the tag slug generator is not documented in API v3? Preferably we'd use the exact same method Ghost uses to convert the tag names into slugs, but their tags API is still experimental...